pkg/common/naming.go

```diff
@@ -50,3 +50,7 @@ func RouteName(workspaceId, endpointName string) string {
 func OAuthProxySecretName(workspaceId string) string {
 	return "proxy-tls-" + workspaceId
 }
+
+func CheRestAPIsConfigmapName(workspaceId string) string {
+	return "che-rest-apis-" + workspaceId
```

Comment:
Consider alligning naming, I prefer workspaceId to be at the beginning a bit more but it's not a strong preference.

![Screenshot_20200416_145007](https://user-images.githubusercontent.com/5887312/79454085-4fc03b00-7ff3-11ea-8aab-bdc8a9f9e677.png)

Reply:
Changed all names in common/naming.go to use workspaceId as prefix.